ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -0,0 +1,467 @@
+        "  metrics_dict = {m[0]: m[1] for m in model_evaluation.metrics.items()}\n",
+        "  df = pd.DataFrame(metrics_dict.items(), columns=[\"Metric\", \"Value\"])\n",
+        "  print(df.to_string(index=False))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nIw1ifPuqcTb"
+      },
+      "source": [
+        "# Run Batch Prediction"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "nT-bZ1autijD"
+      },
+      "source": [
+        "#@markdown ## Create Output BigQuery Dataset\n",
+        "#@markdown First, create a new BigQuery dataset for the batch prediction output in the same region as the batch prediction input dataset. \n",
+        "\n",
+        "import os\n",
+        "from google.cloud import bigquery\n",
+        "\n",
+        "os.environ[\"GOOGLE_CLOUD_PROJECT\"] = PROJECT_ID\n",
+        "\n",
+        "batch_predict_bq_input_uri = \"bq://bigquery-public-data.iowa_liquor_sales_forecasting.2021_sales_predict\"\n",
+        "batch_predict_bq_output_dataset_name = \"iowa_liquor_sales_predictions\"\n",
+        "batch_predict_bq_output_dataset_path = \"{}.{}\".format(PROJECT_ID, batch_predict_bq_output_dataset_name)\n",
+        "batch_predict_bq_output_uri_prefix = \"bq://{}.{}\".format(PROJECT_ID, batch_predict_bq_output_dataset_name)\n",
+        "# Must be the same region as batch_predict_bq_input_uri\n",
+        "client = bigquery.Client()\n",
+        "dataset = bigquery.Dataset(batch_predict_bq_output_dataset_path)\n",
+        "dataset_region = \"US\" # @param {type : \"string\"}\n",
+        "dataset.location = dataset_region\n",
+        "dataset = client.create_dataset(dataset)\n",
+        "print(\"Created bigquery dataset {} in {}\".format(batch_predict_bq_output_dataset_path, dataset_region))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "krKRn9W0xxI2"
+      },
+      "source": [
+        "Run a batch prediction job to generate liquor sales forecasts for stores in Iowa from an input dataset containing historical sales."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "8I8aRjRh6GGG"
+      },
+      "source": [
+        "model.batch_predict(\n",
+        "   bigquery_source=batch_predict_bq_input_uri,\n",
+        "   instances_format=\"bigquery\",\n",
+        "   bigquery_destination_prefix=batch_predict_bq_output_uri_prefix,\n",
+        "   predictions_format=\"bigquery\",\n",
+        "   job_display_name=\"predict-iowa-liquor-sales-automl_1\")"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "24NPJ7nCRchZ"
+      },
+      "source": [
+        "\n",
+        "# Cleaning up\n",
```

Comment:
Hey @ivanmkc, was testing the notebook one more time before merging and found that the clean-up step was malfunctioning. I modified it to get it working so do you mind giving this portion a quick glance before I submit the PR?